contracts/NFTStorefront.cdc

```diff
@@ -124,6 +130,7 @@ pub contract NFTStorefront {
         pub let salePrice: UFix64
         // This specifies the division of payment between recipients.
         pub let saleCuts: [SaleCut]
+        // This lets teams using this contract easily distinguish events that are relevent to them
```

Comment:
What is this comment referring to?

Reply:
I should just remove that. It was in reference to a new field I did not end up being able to include